export/index.html

```diff
@@ -240,6 +252,76 @@ <h2>Message log</h2>
         return new Uint8Array([...s, ...r]);
       }
 
+      /**
+       * Function to verify enclave signature on import bundle received from the server. 
+       */
+       const verifyEnclaveSignature = async (enclaveQuorumPublic, publicSignature, publicKey) => {
+        // Second half is the public key for the enclave quorum encryption key
+        const encryptionQuorumPublic = TURNKEY_SIGNER_ENCLAVE_QUORUM_PUBLIC_KEY.slice(130, 260);
```

Comment:
Wondering if we should commit just what we need to this repo? So basically, instead of:
```
const TURNKEY_SIGNER_ENCLAVE_QUORUM_PUBLIC_KEY = "04ca7c0d624c75de6f34af342e87a21e0d8c83efd1bd5b5da0c0177c147f744fba6f01f9f37356f9c617659aafa55f6e0af8d169a8f054d153ab3201901fb63ecb04cf288fe433cc4e1aa0ce1632feac4ea26bf2f5a09dcfe5a42c398e06898710330f0572882f4dbdf0f5304b8fc8703acd69adca9a4bbf7f5d00d20a5e364b2569"
```

We'd have:
```
const TURNKEY_SIGNER_ENCLAVE_QUORUM_PUBLIC_KEY = "04cf288fe433cc4e1aa0ce1632feac4ea26bf2f5a09dcfe5a42c398e06898710330f0572882f4dbdf0f5304b8fc8703acd69adca9a4bbf7f5d00d20a5e364b2569"
```

This will be less surprising.

Reply:
ya i was going back and forth with this bc one day we might open source the quorum public keys in their concatenated form. but that is def a future problem and a trivial one anyway so i'm down to remove the extra friction that comes with pinning the full key and mysteriously slicing it in half haha (for both here and tkcli which i can update as well)

Reply:
Yeah it's no big deal either way honestly, but it just invites questions to see a slice on public key bytes 👀

Reply:
100% agree haha